src/common/snippets/include/snippets/op/perf_count.hpp

```diff
@@ -11,6 +11,69 @@
 
 namespace ov {
 namespace snippets {
+
+namespace op {
+class PerfCountEnd;
+} // namespace op
+
+namespace utils {
+
+/**
+ * @interface Dumper
+ * @brief Dumper for node debug properties
+ * @ingroup snippets
+ */
+class Dumper {
+public:
+    Dumper() = default;
+    virtual ~Dumper() = default;
+
+    void init(const std::string &params);
+    virtual void update(const op::PerfCountEnd* node) = 0;
+protected:
+    std::map<std::string, std::string> m_debug_params_map;
+    std::string m_params;
+};
+
+/**
+ * @interface ConsoleDumper
+ * @brief Dumper for node debug properties (output: stdout)
+ * @ingroup snippets
+ */
+class ConsoleDumper : public Dumper {
+public:
+    ConsoleDumper() = default;
+    ConsoleDumper(const ConsoleDumper&) = delete;
+    ~ConsoleDumper() override;
+
+    void update(const op::PerfCountEnd* node) override;
+
+private:
+    ov::threading::ThreadLocal<uint64_t> m_accumulation;
+    ov::threading::ThreadLocal<uint32_t> m_iteration;
+};
+
+/**
+ * @interface CSVDumper
+ * @brief Dumper for node debug properties (output: .csv file)
+ * @ingroup snippets
+ */
+class CSVDumper : public Dumper {
+public:
+    CSVDumper(const std::string &csv_path);
```

Comment:
Please pass by value and use std::move to avoid extra copy, like in [this comment](https://github.com/openvinotoolkit/openvino/pull/28378#discussion_r1935276912)

Reply:
Done